client/state/index.js

```diff
@@ -111,9 +115,16 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
 
-export function createReduxStore( initialState = {} ) {
+/**
+ * Create a default set of middleware/enhancers.
+ *
+ * The enhancers included depend on the runtime environment of this code.
+ *
+ * @return { array } The array of enhancers to be configured in a redux store.
+ */
+export function createEnhancers() {
```

Comment:
It does create a list of enhancers though. What it returns does not exist anywhere until it's created by this function.